app/uk/gov/hmrc/mobilehelptosave/services/MilestonesService.scala

```diff
@@ -129,10 +129,10 @@ class HtsMilestonesService[F[_]](
     val hasFirstBonusEstimate              = firstPeriodBonusEstimate > 0
     val hasSecondBonusEstimate             = secondPeriodBonusEstimate > 0
     val firstPeriodBonusPaid               = firstPeriodBonus > 0
-    val within20DaysOfFirstPeriodEndDate   = currentDateInDuration(endOfFirstBonusPeriod, 20)
-    val under90DaysSinceFirstPeriodEndDate = currentDateInDuration(endOfFirstBonusPeriod.plusDays(91), 90)
-    val within20DaysOfFinalEndDate         = currentDateInDuration(endOfSecondBonusPeriod, 20)
-    val dateFormat                         = DateTimeFormatter.ofPattern("d-MMM-yyyy")
+    val within20DaysOfFirstPeriodEndDate   = currentDateInDuration(endOfFirstBonusPeriod, 19)
+    val under90DaysSinceFirstPeriodEndDate = currentDateInDuration(firstPeriodBonusPaidOnOrAfterDate.plusDays(90), 89)
+    val within20DaysOfFinalEndDate         = currentDateInDuration(endOfSecondBonusPeriod, 19)
+    val dateFormat                         = DateTimeFormatter.ofPattern("d-MMMM-yyyy")
```

Comment:
```suggestion
    val dateFormat                         = DateTimeFormatter.ofPattern("d MMMM yyyy")
```

The suggestion Andy made on Slack doesn't have dashes in between the date.
https://hmrcdigital.slack.com/archives/G7L9K3C6S/p1566914756006600